app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,40 @@
+class SessionsController < ApplicationController
+  include JsonWebToken
+
+  # Specify that this controller only responds to JSON format
+  respond_to :json
+
+  def create
+    user = User.find_by(email: params[:user][:email])
```

Comment:
again you could not repeat urself by setting user in a function and calling it in a callback